src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -94,23 +127,38 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime is recommended to use for storing date values in a database.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return localTime.atOffset(ZoneOffset.of("+02:00"));
```

Comment:
make a constant field of "+02:00"

Reply:
We use the zone just in the method, why to do code fat?

Reply:
I the task yes, but you are a developer and maybe would like to reuse it and a bit of clarity isn't bad

Reply:
I see you point, but I can't exept that. Firstly, three variables would use memory for all time long while object is made. for what? As you say, in the task - yes, that's not a lot of, but when it would be prodaction? Secondly, a variable needs just in the method. I got a request from a metntor to chage place in simmilar situation early.  With all my respect to you, I'm not going to chage the consts.

Reply:
As you wish

Reply:
Concerning the memory - constant will consume a tiny amount of memory so you won't feel any difference at all. Plus it's arguable what is better for the performance, if you make a variable local it will create a new one every time method is being called. So let's say we gonna use this method in loop for 100 times - 100 new variables will be created and they won't be collected instantly by garbage collector, as a result memory consumption will be higher. But performance is not the only thing that matter, code readability and reusability is not less important. And in this case declaring your formatters and offset as a constant is better.